Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLink.cs

```diff
@@ -333,6 +373,48 @@ void Start()
             }
         }
 
+        //Set Media Volume display
+        public void SetMediaVolume(float volume) {
+
+            audioMaterial.SetFloat("_MediaVolume", volume);
+
+        }
+
+        //Set Media Time display
+        public void SetMediaTime(float time) {
+
+            audioMaterial.SetFloat("_MediaTime", time);
+
+        }
+
+        /* Set Media Playing display
+        None    0 (0.00)
+        Playing 1 (0.10)
+        Paused  2 (0.20)
+        Stopped 3 (0.30)
+        Loading 4 (0.40)
+        */
+        public void SetMediaPlaying(MediaPlaying playingstate) {
+
+            int state = (int)playingstate;
+            audioMaterial.SetFloat("_MediaPlaying", (float)state / 10);
+
+        }
+
+        /* Set Media Loop display
+        None       0 (0.00)
+        Loop       1 (0.10)
+        LoopOne    2 (0.20)
+        Random     3 (0.30)
+        RandomLoop 4 (0.40)
+        */
+        public void SetMediaLoop(MediaLoop loopstate) {
+
+            int loop = (int)loopstate;
+            audioMaterial.SetFloat("_MediaLoop", (float)loop / 10);
```

Comment:
Why the division by 10? Values >1 should work just fine

Reply:
I wasn't sure that it'd handle more than 1